src/CircleOfFifths.module.css

```
@@ -0,0 +1,65 @@
+
+.circle-slice:focus,
+.circle-slice:focus-visible {
+  outline: none;
+  -webkit-tap-highlight-color: transparent;
+}
+
+.circle-slice-focus-ring {
+  pointer-events: none;
+}
+
+:global(.app-container[data-layout-variant="tablet-stacked"]) .circle-fifths-container,
```

Stylelint (17.7.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)